### ISSUE TITLE
Unable to resize tree-view in a Virtualbox VM

### ISSUE THREAD

**Issue by Asplodzor** [atom/tree-view#513]
I'm running a Kubuntu 14.04 guest VM on a Windows 7 host using Virtualbox. I installed atom on the virtual machine and am unable to resize the tree-view in atom. I have tried to hover my mouse over the tree-view scroll area, and have experimented with moving it back and forth between the scroll area and the start of the text editor pane to its right to no avail. I resize mouse icon never appears, and I am never able to grab onto the side of the tree-view and move it.

**Comment by 50Wliu** [comment]
Atom (more specifically, Chromium <42) currently detects Virtualbox as a touch-friendly device, and so it automatically switches into touch input mode.
Please see atom/atom#6725, atom/atom#6306, atom/atom#6243, and atom/atom#6219 for more details (and atom/atom#6877 which will fix this issue).

**Comment by Asplodzor** [comment]
Thank you for the information! I think that I am experiencing a different problem though. The threads that you linked to say that touch input mode, and I am able to right-click. My mouse behaves like a normal mouse in other ways too, and I do not see any icons appear that indicate touch-sensitivity.

**Comment by 50Wliu** [comment]
@Asplodzor can you see if following the instructions in [this comment](https://github.com/atom/atom/issues/6306#issuecomment-101043098) fixes your problem?

**Comment by Asplodzor** [comment]
@50Wliu thanks for the link. I must have glossed over it before. I tried that fix, but it did not solve my problem. I changed my Pointing Device from USB Tablet to PS/2 Mouse, but after restarting my VM, I see no change in behavior. I am still able to right-click, but am unable to resize any tree-view (or anything else for the matter), and interestingly, I am also unable to grab onto the scroll bar and move it. I can click in the blank area above or below the scroll bar to move it in that direction, but I cannot grab and manipulate the scroll bar itself. That is less of a concern to me than resizing, but it is notable.
